### PR TITLE
Cluster dropdown filtering #204

### PR DESCRIPTION
**Pull request by ErinOC**
This PR addresses (ticket title): #204 

The cluster dropdown persists across tab changes and across page changes.

Note that the fakedata currently doesn't provide the same cluster IDs for cluster-objective-list and cluster-simple-list, so there are no results when using the dropdown filter. Filtering does work when you change the ID in the query string to a cluster ID present in the cluster-objective-list results.

##### Progress checker
- [x ] Polymer

- [ ] Polymer test cases

### CODE REVIEW

File: polymer/src/pages/app/cluster-reporting/planned-action.html [hunk 1]
```diff
@@ -48,6 +57,9 @@
     <template is="dom-if" if="[[_displayHeader(subpage)]]" restamp="true">
       <div class="page-top-content">
         <page-header title="My Planned Action">
+          <div class="toolbar horizontal end-justified layout">
```
Comment (rkrupinski) [inline polymer/src/pages/app/cluster-reporting/planned-action.html]: Do we need layout classes if there's only one child?

File: polymer/src/pages/app/cluster-reporting/planned-action.html [hunk 2]
```diff
@@ -19,14 +20,22 @@
...
 
+      page-header {
+        padding-top: 0;
+      }
+
       .tabs paper-tab {
         text-transform: uppercase;
       }
+
+      .toolbar div {
```
Comment (rkrupinski) [inline polymer/src/pages/app/cluster-reporting/planned-action.html]: Obsolete?

File: polymer/src/redux/reducers/clusters.html [hunk 3]
```diff
@@ -116,5 +120,19 @@
...
 
+    function filterByClusterId(state, action) {
+      if (typeof state === 'undefined') {
+        state = '';
+      }
+
+      switch (action.type) {
+        case App.Constants.SET_FILTER_BY_CLUSTER_ID:
+          return action.id;
+
```
Comment (rkrupinski) [inline polymer/src/redux/reducers/clusters.html]: Lacks `App.Constants.RESET` branch.

File: polymer/src/pages/app/cluster-reporting/planned-action.html (hunk 1 above)
Comment (ErinOC) [inline polymer/src/pages/app/cluster-reporting/planned-action.html]: To make it end-justified, yes.

File: polymer/src/pages/app/cluster-reporting/planned-action.html (hunk 2 above)
Comment (ErinOC) [inline polymer/src/pages/app/cluster-reporting/planned-action.html]: Removed.

File: polymer/src/redux/reducers/clusters.html (hunk 3 above)
Comment (ErinOC) [inline polymer/src/redux/reducers/clusters.html]: Added, thanks.

**Pull request merged**